src/latte/kernel/syntax.clj

```diff
@@ -305,9 +305,12 @@
  (alpha-norm '[x (λ [x ✳] (test x y [x z]))])
  => '[x (λ [_1 ✳] (test _1 y [_1 z]))])
 
-(defn alpha-eq? [t1 t2]
-  (= (alpha-norm t1)
-     (alpha-norm t2)))
+(defn alpha-eq?
+  "Redefine as a multi-arity fn if the variadic implementation is too slow"
+  #_([t1] true)
+  #_([t1 t2] (= (alpha-norm t1) (alpha-norm t2)))
+  #_([t1 t2 & more] (apply = (map alpha-norm (apply list t1 t2 more))))
+  [t1 t2 & more] (apply = (map alpha-norm (apply list t1 t2 more))))
```

Comment:
this is ok but I don't know if it would be interesting beyond tests... (and I don't want tests to impact the code, only the converse).